assets/quick-order-list.js

```diff
@@ -334,8 +334,13 @@ class QuickOrderList extends HTMLElement {
   }
```

Comment:
Looks good 👌 I think the spacing is a bit far now. But ill let @edmund-teh and @nicklepine take a look! https://screenshot.click/21-41-ihm3k-fnjbu.png

Reply:
Thanks, yes is there any way to reduce some of the vertical spacing to be closer to what it used to be
reference pre-this pr: https://screenshot.click/21-11-f021j-blp5l.png